lib/automate_liveness_agent/main.rb

```diff
@@ -111,6 +111,7 @@ def send_keepalives
       if config.scheduled_task_mode
         a.update
       else
+        Process.daemon
```

Comment:
This might impact the init script, since our pid will change from the one we started with.